test/util/constants.go

```diff
@@ -43,6 +43,7 @@ const (
 	StartUpTimeout     = 5 * time.Minute
 	ConsistentDuration = time.Second
 	Interval           = time.Millisecond * 250
+	ConsistentInterval = 10 * time.Millisecond
```

Comment:
Thanks for defining it here.
```suggestion
	ShortInterval = 10 * time.Millisecond
```
nit, to align with the naming convention for `Timeout`